.readthedocs.yaml

```diff
@@ -3,6 +3,6 @@ version: 2
 formats: all
 
 python:
-    install:
-        - requirements: doc-requirements.txt
-        - path: .
+  install:
+    - path: .
```

Comment:
I think this small changeset already landed in main per a James branch Friday. might need a `git rebase` to make sure something else in James' doc branches doesn't adversely affect your changes in rendering.